src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -195,7 +196,7 @@ private static void RegisterCustomNodeInstanceForUpdates(Function node, CustomNo
                 node.Category = info.Category;
             };
             workspace.InfoChanged += infoChangedHandler;
-            node.Disposed += () =>
+            node.Disposed += delegate
             {
```

Comment:
Not sure why you're changing this to delegate - it's much more readable as a lambda.